workitem/workitem_repository.go

```diff
@@ -110,14 +111,54 @@ func (r *GormWorkItemRepository) Load(ctx context.Context, spaceID uuid.UUID, wi
 	return ConvertWorkItemStorageToModel(wiType, wiStorage)
 }
 
+// LookupIDByNamedSpaceAndNumber returns the work item's ID for the given owner name, space name and item number
+// returns NotFoundError, ConversionError or InternalError
+func (r *GormWorkItemRepository) LookupIDByNamedSpaceAndNumber(ctx context.Context, ownerName, spaceName string, wiNumber int) (*uuid.UUID, *uuid.UUID, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "LookupIDByNamedSpaceAndNumber"}, time.Now())
```

Comment:
I think @hectorj2f would want me to point out that we want to have lower cases here. But I'm not sure.